markdown/3.x/en/documentation/upgrade-guide.md

```diff
@@ -447,7 +468,33 @@ tagCloudLink
 ffw-tagCloudLink
 ```
 
+
+### Replacement of identifiers
```

Comment:
```suggestion
### Replacement of HTML `id` properties
```

I think _HTML id properties_ is more clear than _identifiers_, isn't it? (same would apply to next comment in the change)

Reply:
This is not relevant any more, as I removed entire section due to your comment.